src/loginServices.js

```diff
@@ -110,15 +115,50 @@ export function getBindings(okapiUrl, store, tenant) {
     });
 }
 
+function getErrorMessage(error = {}) {
+  const { code = 'default.error', parameters } = error;
+  const values = parameters ?
+    parameters.reduce(
+      (res, key) => {
+        res[key.key] = key.value;
+        return res;
+      }, {}
+    ) : {};
+
+  return <FormattedMessage id={`stripes-core.loginServices.login.${code}`} values={values} />;
+}
+
+function getPasswordValidationMessages(errors) {
+  return errors.map(element => (
+    <li key={`${element.code}-${element.type}`}>
+      { getErrorMessage(element)}
+    </li>
+  ));
+}
+
+function parseErrors(json) {
+  const { errorMessage } = json;
+  try {
+    const { errors } = JSON.parse(errorMessage);
+
+    return errors.length > 1 ? (
+      <ul style={styles.multipleErrorsUl}>{getPasswordValidationMessages(errors)}</ul>
```

Comment:
Could we move the style information to a `.css` file and use `className=""` instead of inline styles?